src/Security.php

```diff
@@ -15,7 +15,7 @@ class Security
      * @param  string $xml
      * @throws Exception\RuntimeException If entity expansion or external entity declaration was discovered.
      */
-    protected static function heuristicScan($xml)
+    protected static function heuristicScan(string $xml): void
```

Comment:
That's a BC break on a non-final class. Feel free to add `/** @final **/` to the class doc block to signal future intent, but this still can't be changed in a minor release.

Reply:
Also, you may want to add the BC Checker to CI like this: https://github.com/laminas/laminas-filter/pull/135/files